onnxruntime/contrib_ops/cuda/moe/ft_moe/moe_kernel.cu

```diff
@@ -47,6 +47,8 @@
 #include "cub/util_type.cuh"
 #endif
 
+#include "core/providers/cuda/cu_inc/common.cuh"
```

Comment:
The included file is defined in onnxruntime::cuda namespace. This file use ort_fastertransformer namespace.
Try add `using namespace onnxruntime::cuda` to avoid build error.

Reply:
seems there're pipeline errors. let me revert to the initial one/add a todo to unblock the package pipeline.